torch/optim/swa_utils.pyi

```diff
@@ -1,16 +1,27 @@
-from typing import Any, Callable, Iterable, Union
+from typing import Any, Callable, Iterable, List, Tuple, Union
 
 from torch import device, Tensor
 from torch.nn.modules import Module
 from .lr_scheduler import _LRScheduler
 from .optimizer import Optimizer
 
+PARAM_LIST = Tuple[Tensor, ...] | List[Tensor]
+
+def get_ema_multi_avg_fn(decay: float) -> Callable[[PARAM_LIST, PARAM_LIST, Any], None]: ...
+
+def get_swa_multi_avg_fn() -> Callable[[PARAM_LIST, PARAM_LIST, int], None]: ...
+
+def get_ema_avg_fn(decay: float) -> Callable[[Tensor, Tensor, Any], Tensor]: ...
+
+def get_swa_avg_fn() -> Callable[[Tensor, Tensor, int], Tensor]: ...
+
 class AveragedModel(Module):
     def __init__(
         self,
         model: Module,
         device: Union[int, device] = ...,
         avg_fn: Callable[[Tensor, Tensor, int], Tensor] = ...,
```

Comment:
Looking at the code (looking for `self.avg_fn(` and `self.multi_avg_fn(`) shows that the third arg should really be a scalar Tensor and not an int.